core/docs/spark-profiling-tool.md

```diff
@@ -380,7 +380,8 @@ This displays several data sections.
    and the shuffle read time as reported by Spark. The shuffle time works for both CPU and GPU
    tasks, but "buffer time" only is reported for GPU accelerated file reads.
    3. Red is the semaphore wait time. This is the amount of time a task spent waiting to get access
-   to the GPU. This only shows up on GPU tasks when DEBUG metrics are enabled. It does not apply to
+   to the GPU. In versions of the plugin prior to 23.04 this would only show up on GPU tasks when
+   DEBUG metrics are enabled. On 23.04 and above it is always on. It does not apply to
```

Comment:
Here, "23.04" refers to Rapids-plugin jar version, right? The user could mistakenly expect to see same information when he runs  spark-rapids-tools 23.04+ on eventlogs generated by older RAPIDS plugin.